UnityProject/Assets/Scripts/Util/ScriptsUtilMonoBehaviourSingleton.cs

```diff
@@ -14,6 +14,11 @@ public virtual void Awake()
 	}
 
 
+	public virtual void Start()
+	{
+		Instance = this as T;
+	}
```

Comment:
isnt it already set at Awake?

Reply:
yeah, but that reference doesn't stick for some reason and just becomes null